ext/uri/tests/rfc3986/modification/userinfo_success_existing.phpt

```diff
@@ -0,0 +1,19 @@
+--TEST--
+Test Uri\Rfc3986\Uri omponent modification - userinfo - changing an existing one
```

Comment:
This description matches the file name, but not what is being tested: There is no existing userinfo.